### ISSUE TITLE
Cannot send

### ISSUE THREAD

**Issue by malins** [dark-eye/com.darkeye.chatGPT#22]
This widget is no longer working, I cannot send any text, neither with the "Send on enter" feature, nor with the UI button.

**Comment by eletromarlon** [comment]
Sim! Também não consigo utilizar há dias.

**Comment by marceymard** [comment]
Same here. About a month ago, the widget stopped sending request when either pressing enter or clicking the send button.

Plasma version 5.27.5
Frameworks version 5.103.0
Qt 5.15.8

**Comment by dark-eye** [comment]
Yes this seem  to be an issue with  ChatGPT  thinking  we are  bots (ironically).
I'm a little  busy  currently  so  i  could get  my self  to  workaround it. 

(some time it  can start  working again if you  reload the site)
  
Sorry  once  i'll  get  enough free time  i'll work on solving it
